presidio-analyzer/analyzer/pattern_recognizer.py

```diff
@@ -50,14 +52,20 @@ def __init__(self, supported_entity, name=None, supported_language='en',
     def load(self):
         pass
 
-    def analyze(self, text, entities):
+    # pylint: disable=unused-argument
+    def analyze(self, text, entities, nlp_artifacts=None):
         results = []
 
         if self.patterns:
             pattern_result = self.__analyze_patterns(text)
 
             if pattern_result:
```

Comment:
Should also check if no context is supported.

Reply:
👍